src/net.c

```diff
@@ -997,6 +997,25 @@ coap_send(coap_session_t *session, coap_pdu_t *pdu) {
     goto error;
   }
 
+#if !COAP_DISABLE_TCP
+  if (COAP_PROTO_RELIABLE(session->proto) && !session->csm_block_supported) {
+    /*
+     * Need to check that this is not sending any block options as the
```

Comment:
"that this is not": here is a word missing ("instance", "peer"?)

Reply:
Text corrected to include instance.